tutorials/navigation/navigation_connecting_navmesh.rst

```diff
@@ -0,0 +1,62 @@
+.. _doc_navigation_connecting_navmesh:
+
+Connecting NavigationMeshes
+===========================
+
+Different NavigationMeshes are automatically merged by the NavigationServer 
+when at least two vertex positions of one edge exactly overlap.
+
+.. image:: img/navigation_vertex_merge.png
+
+The same is true for multiple NavigationPolygon resources. As long as their 
+outline points overlap exactly the NavigationServer will merge them.
+NavigationPolygon outlines must be from different NavigationPolygon resources to connect.
+
+Overlapping or intersecting outlines on the same NavigationPolygon 
+will fail the navigation mesh creation. Overlapping or intersecting 
+outlines from different NavigationPolygons will often fail the 
+navigation region connections on the NavigationServer and should be avoided.
```

Comment:
Feels like a work may be missing hereß "fail _at_ the connections"? I dunno, maybe I'm just missing something :)

Reply:
... will often fail to create the navigation region `edge connections` on the NavigationServer ...

The `edge connections` are their own thing build by the server behind the scene. Merges the free edges of a navigationregion navmesh with other free navigationregion navmesh edges by proximity / angle.